pkg/restapi/operation/operations.go

```diff
@@ -65,16 +67,27 @@ func (o *Operation) createKeystoreHandler(rw http.ResponseWriter, req *http.Requ
 		return
 	}
 
-	var config models.KeystoreConfiguration
+	var config keystore.Configuration
 	err = json.Unmarshal(requestBody, &config)
 	if err != nil {
 		writeErrorResponse(rw, http.StatusBadRequest, fmt.Sprintf(receivedInvalidConfiguration, err))
 		return
 	}
 
-	keystoreID, err := o.createKeystore()
+	if config.Sequence != 0 {
```

Comment:
why even have `Sequence` as an input anyway?

Reply:
I added it here to be compliant with the way how digitalbazaar deals with a configuration in [their code](https://github.com/digitalbazaar/bedrock-kms/blob/3ae50ddf777487bcdf086bcc06312d70339d5863/lib/keystores.js#L59). Given that they are main contributors to the [spec](https://w3c-ccg.github.io/webkms/), most probably, this will land into future revisions of the spec. In our case, the value itself is not required but if it is specified - it should be 0.